scripts/label_validator.sh

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env bash
+
+# standard bash error handling
+set -o nounset  # treat unset variables as an error and exit immediately.
+set -o errexit  # exit immediately when a command fails.
+set -E          # must be set if you want the ERR trap
+set -o pipefail # prevents errors in a pipeline from being masked
+
+GITHUB_ORG="kyma-project"
+
+# From Github API Docs why using API for Issue.
```

Comment:
This comment needs to be clarified. Do you mean `From Github API Docs on why use API for Issue`?